README.md

```diff
@@ -139,38 +139,57 @@ await waitForActions(store, [
 ]);
 ```
 
-### Example #3: function (advanced use cases)
+### Example #3: custom action matcher (advanced use cases)
```

Comment:
Don't we think that we could simplify the readme a bit now? We could have:

# Installation

`$ npm i... `

# Usage

- Example with string actions
- Example with object actions

## Options

- timeout
- matcher

Here we provide info about the matcher implementation, built-in matchers, with examples.